cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -307,48 +303,61 @@ <h3>Data table</h3>
+                                           cf-icon__after
+                                           cf-icon-down">
+                                Download chart
+                            </button>
+                            <ul class="chart-menu_options">
+                                <li>PNG</li>
+                                <li>SVG</li>
+                                <li>JPEG</li>
+                                <li>Print chart</li>
+                            </ul>
+                        </div>
                         <figure class="data-enabled loading">
                             <div id="chart" class="chart-area"></div>
                             <figcaption class="chart-caption">
                                 <div class="caption-title">
                                     Interest rates for your situation
                                 </div>
                                 <div class="rc-data-link">
-                                    <a href="#about" class="u-link-underline">About our data source</a>
+                                    <a href="#about" class="u-link-underline">Read about our data source</a>
                                 </div>
                             </figcaption>
                         </figure>
 
                         {# Alert Shown when there are no results. #}
                         <div id="chart-result-alert"
-                             class="result-alert chart-alert u-hidden"
+                             class="result-alert
+                                    m-notification
+                                    m-notification__error"
                              role="alert">
-                          <p class="alert">
-                            <strong>We're sorry!</strong>
-                            Based on the information you entered,
-                            we don't have enough data to display results.
-                          </p>
-                          <p class="point-right">
-                            Change your settings
-                          </p>
-                          <p>
-                            <a id ="reload-link" class="defaults-link" href="">
-                              Or, revert to our default values
-                            </a>
-                          </p>
+                            <span class="m-notification_icon cf-icon"></span>
+                            <div class="m-notification_content">
+                                <div class="h4 m-notification_message">We're sorry!</div>
+                                <div class="h4 m-notification_explanation">
+                                    Based on the information you entered,
+                                    we don't have enough data to display results.
+                                    Change your settings or
+                                    <a id ="reload-link" class="defaults-link" href="">
```

Comment:
Remove space between attribute and equal sign `id="reload-link"`

Reply:
fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/805de5edd2f3f85ba299ecef445c033fc1e117a9.